index.js

```diff
@@ -51,10 +52,26 @@ app.use("/preference", preferenceRoutes);
 app.use("/message", messageRoutes);
 
 // Swagger routes
-app.use('/docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument))
+app.use("/docs", swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
 // Health check for GCP stuff
-app.use("/health", (req, res) => { res.sendStatus(200); } );
+app.use("/health", (req, res) => {
+  res.sendStatus(200);
+});
+
+// Cron job for weekly assignment creation @ 12AM CST on Monday
+const AssignmentService = require("./src/services/assignment.service");
+const job = new cron.CronJob(
+  "0 0 1 * * 1",
```

Comment:
`0 0 1 * * 1` is 1 a.m on Mondays. It should probably be `0 0 0 * * 0` for 12 a.m. on Sundays